HackYourFuture-CPH/simply-name.it
# install express-async-errors

# Description

Install express-async-errors

Fixes #112

# How to test?

Install and verify that errors thrown in async methods a propagated to the middleware error handling function in app.js

# Checklist
